example/package.json

```diff
@@ -18,5 +18,10 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "bootstrap": "^4.1.0",
+    "jquery": "^3.3.1",
```

Comment:
Why are we including these 3 dependencies?

Reply:
@AaronCoplan They're all required for bootstraps javascript functionality such as popovers and dropdown triggers, ideally, I imagine we will want to rebuild these ourselves but it's the quickest way of adding the functionality in the short term

Reply:
Are you sure we need all three?  Wouldn't listing bootstrap as a dependency automatically include the other two because it recursively fetches its dependencies?  If not, no worries

Reply:
Unfortunately not they're peerDependencies in the bootstrap package so we have to bring them ourselves :(